src/core/api/search.js

```diff
@@ -57,25 +56,12 @@ export default function search(
     _filters.clientApp = 'firefox';
   }
 
-  // Themes are cross-platform and searching for them with an operatingSystem
-  // filter will result in no results, so we delete it for now.
-  // Can be deleted once https://github.com/mozilla/addons-server/issues/6206
```

Comment:
Note: this is only fixed on dev right now, so make sure to test this against dev or it will error.